bin/collator/src/parachain/service.rs

```diff
@@ -596,6 +614,9 @@ pub struct AdditionalConfig {
 
     /// Soft deadline limit used by `Proposer`
     pub proposer_soft_deadline_percent: u8,
+
+    /// Hardware benchmarks score
```

Comment:
_score_ typo?

Reply:
Not sure what I saw before lol, but it wasn't score. But I even wrote it 😂 

Time for OOO xD